app/models/ems_event.rb

```diff
@@ -146,6 +146,34 @@ def self.process_vm_in_event!(event, options = {})
         event[:vm_name] ||= vm.name
       end
     end
+
+    event_type = event[:event_type]
+    # both event types are Rhev specific
+    if (event_type == 'USER_ADD_VM' || event_type == 'USER_ADD_VM_FINISHED_SUCCESS') && event[:vm_or_template_id].nil?
```

Comment:
>> So it does only work if something already exists on our side in the path, like the host.

> Do you mean it only works if they exist in the database already? A targeted refresh should work even if none of the associated inventory was in the database yet AFAIK

No, I mean, the root of the inventory tree has to exist. As I understand this whole thing here, is they have to create the vm during the event in order to do a targeted refresh on the vm.

In vmware you would do a targeted refresh on the Host that that new vm is on